app/src/pages/inside/common/durationBlock/utils.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 EPAM Systems
```

Comment:
We prefer to update the year in new files.
So, that is why I recommend you to update it to 2023

Reply:
Done.